project_national/fact_sheets_reference.yml

```diff
@@ -0,0 +1,104 @@
+schema_version: '0.3'
+os_version: 3.5.1
+os_sha: 22e1db7be5
+buildstock_directory: ../ # Relative to this file or absolute
+project_directory: project_national # Relative to buildstock_directory
+output_directory: national_baseline
+# weather_files_url: https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip
+weather_files_path: /shared-projects/buildstock/weather/BuildStock_TMY3_FIPS.zip
+
+sampler:
+  type: precomputed
+  args:
+    sample_file: ../buildstock_10k_20230520.csv # relative to file
+
+baseline:
+  n_buildings_represented: 136569411 # American Community Survey 2019 5-year, B25001, does not include AK, HI, and territories
+  skip_sims: true
+
+eagle:
+  n_jobs: 8
+  minutes_per_sim: 4
+  account: abctypol
+  postprocessing:
+    time: 30
+    n_workers: 3
+  sampling:
+    time: 5
+
+aws:
+  job_identifier: fs_ref
+  s3:
+    bucket: largeee
+    prefix: test_runs
+  region: us-west-2
+  use_spot: true
+  batch_array_size: 10000
+  notifications_email: lliu2@nrel.gov
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/cambium/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: hourly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
```

Comment:
Do we want the QOI report? Probably but I don't know that we've discussed this.

Reply:
As QOI is an annual output, I think it's smart to at least have them in our results.